trade-finance-manager-api/src/v1/controllers/deal-cancellation.controller.test.ts

```diff
@@ -0,0 +1,42 @@
+import { ObjectId } from 'mongodb';
+import httpMocks from 'node-mocks-http';
+import api from '../api';
+import { updateDealCancellation, UpdateDealCancellationRequest } from './deal-cancellation.controller';
+
+jest.mock('../api');
+
+const dealCancellationUpdate = {
+  reason: 'test reason',
+  bankRequestDate: 1794418807,
+  effectiveFrom: 1794418808,
+};
+
+const mockDealId = new ObjectId();
+const mockUserId = new ObjectId();
+
+describe('controllers - deal cancellation', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
+  describe('PUT - updateDealCancellation', () => {
```

Comment:
Could you add tests for error handling & returning the correct thing when successful

Reply:
Yep added!